plugins/modules/one_vm.py

```diff
@@ -785,7 +878,8 @@ def get_vm_info(client, vm):
         'uptime_h': int(vm_uptime),
         'attributes': vm_attributes,
         'mode': permissions_str,
-        'labels': vm_labels
+        'labels': vm_labels,
+        'updateconf': updateconf
```

Comment:
```suggestion
        'updateconf': updateconf,
```
Will make the next diff nicer when another entry is added :)